backend/test/deal-observer.test.js

```diff
@@ -107,6 +107,64 @@ describe('deal-observer-backend', () => {
     await storeBlockEvent(data)
     assert.strictEqual(await countStoredActiveDeals(pgPool), 2n)
   })
+
+  it('simultaneously processes claims for a piece stored twice in the same sector', async () => {
```

Comment:
In my understanding, "it simultaneously processes" means one query storing two deals, while "it serially processes" means one query per deal. Is that what you had in mind, @juliangruber?

If that's the case, then you need to swap the test titles, @NikolasHaimerl, since the first test is executing two SQL queries.

Reply:
Yes that's what I had in mind